.doc_gen/metadata/sfn_metadata.yaml

```diff
@@ -503,5 +635,4 @@ sfn_Scenario_GetStartedStateMachines:
   services:
     sfn: {CreateActivity, CreateStateMachine, DeleteActivity, DeleteStateMachine, DescribeExecution,
           DescribeStateMachine, GetActivityTask, Hello, ListActivities, ListStateMachines,
```

Comment:
What is "Hello"? Is that correct in this list?

Reply:
I didn't add Hello to that list. As you can see, it was there already.

Reply:
I got confirmation that it shouldn't be there. You can remove it in this PR, or we can address it in a separate PR.